docs/tutorials/field_type/7_add_content_and_edit_views.md

```diff
@@ -1,275 +1,116 @@
 # Add content and edit views
 
-To able to add and edit Content Item with our Field Type using PlatformUI, we need to create an edit view.
-The content view will also be needed to display correct information when viewing the Content Item in the "Content structure" tab in PlatformUI.
+To able to add and edit Content Item with our Field Type using PlatformUI, we will implement a FormMapper with FieldValueFormMapperInterface. 
+Also DataTransformer is needed in order to correctly transform our Value Object into single inputbox.
```

Comment:
`DataTransformer is also needed` ... `single input box`, or even better `input field` because this is what you use later on.